src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -466,6 +467,26 @@ def list_subvolumes(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def subvolume_exists(self, **kwargs):
+        volname = kwargs['vol_name']
+        groupname = kwargs['group_name']
+        ret = 0, "", ""
+        volume_exists = False
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                volume_exists = True
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    res = group.has_subvolumes()
+                    if res:
+                        ret = 0, "Subvolume(s) exist", ""
+                    else:
+                        ret = 1, "No subvolume exist", ""
```

Comment:
s/No subvolume exist/no subvolume exists/ ? Matching the case with the success message ?

Reply:
Also same comment as in 'subvolume_group_exists'